### PR TITLE
chore: move to turbo

### PR DESCRIPTION
**Pull request by mPaella**
## Description
should have migrated to turbo long ago

## Test plan

tested locally, plus ci

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,7 @@
         "clean:node_modules": "pnpm dlx rimraf \"**/node_modules\" --glob",
         "lint": "biome check --diagnostic-level=error --max-diagnostics=200",
         "lint:fix": "biome check --write",
-        "publish": "lerna publish from-package",
-        "test": "lerna run test"
+        "test:vitest": "pnpm turbo test:vitest"
```
Comment (AlbertoElias) [inline package.json:19]: why rename? to maintain consistency with nextjs app?

File: package.json [hunk 2]
```diff
@@ -16,8 +16,7 @@
         "clean:node_modules": "pnpm dlx rimraf \"**/node_modules\" --glob",
         "lint": "biome check --diagnostic-level=error --max-diagnostics=200",
         "lint:fix": "biome check --write",
-        "publish": "lerna publish from-package",
```
Comment (AlbertoElias) [inline package.json:19]: was this not used at all?

File: apps/wallets/smart-wallet/next/turbo.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://turbo.build/schema.json",
+    "extends": ["//"],
+    "tasks": {
+        "build": {
+            "dependsOn": ["^build"],
+            "inputs": [".env", ".env.local"],
```
Comment (AlbertoElias) [inline apps/wallets/smart-wallet/next/turbo.json:7]: should these applications also use tsup or not worth it? (not for this PR)

**Review by AlbertoElias — APPROVED** [review]
looks good!

File: turbo.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://turbo.build/schema.json",
+    "tasks": {
+        "build": {
+            "outputLogs": "new-only",
+            "dependsOn": ["^build"]
+        },
+        "test:circular": {},
```
Comment (mPaella) [inline turbo.json:8]:
```suggestion
```

File: package.json (hunk 1 above)
Comment (mPaella) [reply package.json:19]: yeah, and just "test" is pretty ambiguous

File: package.json (hunk 2 above)
Comment (mPaella) [reply package.json:19]: correct, all moved to changesets

File: apps/wallets/smart-wallet/next/turbo.json (hunk 3 above)
Comment (mPaella) [reply apps/wallets/smart-wallet/next/turbo.json:7]: i dont even think its possible to use tsup for a nextjs/create react app

**Pull request merged**